src/NServiceBus.Gateway/Deduplication/LegacyDeduplicationWrapper.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace NServiceBus.Gateway.Deduplication
```

Comment:
Nit: Should be `NServiceBus.Gateway` according to https://github.com/Particular/PlatformArchitecture/blob/master/documentation/cross-cutting-concepts.md#nservicebus-rules ?